packages/fluentui/react-bindings/src/icons/createSvgIcon.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { Extendable } from '@fluentui/styles';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+
+import { StylesContextValue } from '../styles/types';
+import useStyles from '../hooks/useStyles';
+import getUnhandledProps from '../utils/getUnhandledProps';
+import { SvgIconCreateFnParams, SvgIconProps, withSafeTypeForSpan } from './types';
+
+export const SvgIconClassName = 'ui-icon';
+export const SvgIconDisplayName = 'SvgIcon';
+
+export const SvgIconHandledProps: (keyof SvgIconProps)[] = [
+  'alt',
+  'aria-label',
+  'bordered',
+  'className',
+  'circular',
+  'children',
+  'design',
+  'disabled',
+  'outline',
+  'size',
+  'rotate',
+  'styles',
+  'variables',
+  'xSpacing',
+];
+
+const createSvgIcon = <TProps extends SvgIconProps>({
+  svg,
+  displayName,
+  handledProps = SvgIconHandledProps,
+}: SvgIconCreateFnParams) => {
+  const Component: React.FC<TProps> & { handledProps: string[] } = (props: Extendable<SvgIconProps>) => {
```

Comment:
If this will be always `span`:

```suggestion
  const Component: React.FC<React.HTMLAttributes<HTMLSpanElement> & TProps> & { handledProps: string[] } = (props) => {
```